apollo-android-support/build.gradle

```diff
@@ -48,3 +48,4 @@ dependencies {
 }
 
 apply from: rootProject.file('gradle/gradle-mvn-push.gradle')
```

Comment:
Do we need both?

Reply:
Yes, snapshots still publishing to sonatype